addons/portal/static/src/scss/portal.scss

```diff
@@ -362,8 +362,15 @@ img, .media_iframe_video, .o_image {
             @include o-position-sticky($top: $spacer * 5);
             z-index: 1;
         }
-    }
-}
+        h2 {
+            word-wrap: break-word !important;
```

Comment:
`word-break: break-word` is not enough ? Removing this line doesn't change anything...